packages/models-library/src/models_library/api_schemas_webserver/projects_ports.py

```diff
@@ -0,0 +1,45 @@
+from typing import Any
+
+from models_library.projects_nodes_io import NodeID
+from pydantic import BaseConfig, BaseModel, Extra, Field
+
+
+class _InputSchemaConfig(BaseConfig):
```

Comment:
Eventually these configs should be the same as those in `packages/models-library/src/models_library/api_schemas_webserver/_base.py`.  
Unfortunately the web-api is not consistent enough for this  but could you please check if you can use the ones in _base? Otherwise it is ok

Reply:
I removed this code for another PR, hence resolving this conversation